### PR TITLE
Debug script tweaks.

### PR DESCRIPTION
**Pull request by PromoFaux**
_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

**By submitting this pull request, I confirm the following (please check boxes, eg [X]):**

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] I have checked that [another pull request](https://github.com/pi-hole/pi-hole/pulls) for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**How familiar are you with the codebase?:**
- [x] 10 (very familiar)

---

Fix shebang line broken in #831

Remove root check and references to ${SUDO}

Check for /etc/pihole/setupVars.conf and use piholeInterace in IPv6 checks as suggest by @jacobsalmela https://github.com/pi-hole/pi-hole/pull/831#issuecomment-255606854

### CODE REVIEW

**Comment by PromoFaux** [comment]
Engage

File: advanced/Scripts/piholeDebug.sh [hunk 1]
```diff
@@ -33,32 +33,15 @@ cat << EOM
 ::: and optionally upload the generated log to a unique and random directory on
 ::: Termbin.com. NOTE: All log files auto-delete after 1 month and you are the only
 ::: person who is given the unique URL. Please consider where you post this link.
-::: 
+:::
 EOM
 
-
```
Comment (dschaper) [inline advanced/Scripts/piholeDebug.sh]: Why did this leave? Because it's only fired from `pihole -d` and `pihole` is a sudo script?

**Comment by dschaper** [comment]
Approved

File: advanced/Scripts/piholeDebug.sh (hunk 1 above)
Comment (PromoFaux) [inline advanced/Scripts/piholeDebug.sh]: To get in line with the rest of the scripts (See #814)m `pihole` (well, and `basic-install.sh`) will be the only script to have the root check, as everything else is called by it. No need to keep checking for elevation in each script called by `pihole`! 

It also means that script's **must** be called using `pihole`, although if you wanted you could run `sudo /opt/pihole/somescript.sh`

**Comment by jacobsalmela** [comment]
Approved

**Pull request merged**